### PR TITLE
WIP: Add serf

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/staged-recipes/issues/1663
Requires https://github.com/conda-forge/staged-recipes/pull/2815

This is not yet complete. So is probably not worthy of a review yet. Just putting this up here now as a reminder to come back to it.

This packages `serf`, which is a requirement of `svn`. It requires `apr` to be built.

Todo:

* [ ] Handle Windows support where `scons` can't.

### CODE REVIEW

**Comment by jakirkham** [comment]
@msarahan, what is the standard way of handling something like `scons` on Windows these days? I know `scons` is Python 2 only, but this library should build fine with any VS version. Is there a good way to ensure this can be built against all VC runtimes?

File: recipes/serf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  description: |
+    Serf is a simple web server. It is a development tool to be used during the
+    development of rich internet applications.It is not intended for deployment.
+  dev_url: https://svn.apache.org/repos/asf/serf/
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - mingwandroid
+    - nehaljwani
```
Comment (jakirkham) [inline recipes/serf/meta.yaml]: Have gone ahead and added @mingwandroid and @nehaljwani as they authored the recipe in `defaults`. Please let me know if that is ok.

Comment (nehaljwani) [reply recipes/serf/meta.yaml]: I am fine with it.

**Comment by jakirkham** [comment]
Seems the Linux failure is the same as issue ( https://github.com/conda/conda-build/issues/3165 ).

**Comment by jakirkham** [comment]
Superseded by PR ( https://github.com/conda-forge/staged-recipes/pull/11714 ). The feedstock is ( https://github.com/conda-forge/serf-feedstock ).

**Pull request closed without merging**